SimpleXML/SimpleXML.php

```diff
@@ -329,7 +329,7 @@ public function count () {}
  * <b>TRUE</b> if <i>ns</i> is a prefix, <b>FALSE</b> if it's a URI;
  * defaults to <b>FALSE</b>.
  * </p>
- * @return SimpleXMLElement an object of class SimpleXMLElement with
+ * @return mixed an object of class SimpleXMLElement with
```

Comment:
`SimpleXMLElement|false`.

Some PhpStorm meta magic should be applied to gather return type from `$className` parameter.